File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
class Solution(object):
    def minBitFlips(self, start, goal):
        result=start^goal
        i=0
        while result > 0:
            i+=result&1
            result>>=1
        return i
        # return bin(start^goal).count('1') --> one line answer <--
        

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.java
class Solution {
    public int minBitFlips(int start, int goal) {
        int result=start^goal;
        int i=0;
        while(result>0){
            i+=result&1;
            result>>=1;
        }
        return i;
        //return Integer.bitCount(start^goal); --> single line answer <--
    }
}